AsyncDisplayKit/ASTableView.mm

```diff
@@ -1509,7 +1510,7 @@ - (void)didLayoutSubviewsOfTableViewCell:(_ASTableViewCell *)tableViewCell
   // Normally the content view width equals to the constrained size width (which equals to the table view width).
   // If there is a mismatch between these values, for example after the table view entered or left editing mode,
   // content view width is preferred and used to re-measure the cell node.
-  if (contentViewWidth != constrainedSize.max.width) {
+  if (CGSizeEqualToSize(node.calculatedSize, CGSizeZero) == NO && contentViewWidth != constrainedSize.max.width) {
```

Comment:
Just checking - do we need to make the same change in ASCollectionView?  There isn't one as part of the diff, so maybe it was already correct or doesn't have the same code.